vnext/Microsoft.ReactNative/TurboModulesProvider.cpp

```diff
@@ -85,9 +86,16 @@ class TurboModuleImpl : public facebook::react::TurboModule {
       ReactModuleProvider reactModuleProvider)
       : facebook::react::TurboModule(name, jsInvoker), m_moduleBuilder(winrt::make<TurboModuleBuilder>(reactContext)) {
```

Comment:
>, [](start = 53, length = 1)

(nit) could we make single initialization per line for readability? Or our auto formatters don't like it?

Reply:
We use automated code formatting. We cannot control how it looks.